pkgs/development/tools/zigmod/default.nix

```diff
@@ -0,0 +1,163 @@
+    }
+    {
+      url = "https://github.com/nektro/arqv-ini";
+      rev = "1a9b2e90379895e197893b6e19c93bd213ad36e6";
+      sha256 = "sha256-4ffAFedGldAM9aF/8HrHINHlSEddyeIQap8ez1MqrmQ=";
+    }
+    {
+      url = "https://github.com/nektro/iguanaTLS";
+      rev = "bd003aa75c8bc00e8a61a1be575b9305249dc749";
+      sha256 = "sha256-fVfyQvK+3DxupgSfmf0VbjmS8sIN4hNhbAPtrv+SujY=";
+    }
+    {
+      url = "https://github.com/nektro/zig-ansi";
+      rev = "d4a53bcac5b87abecc65491109ec22aaf5f3dc2f";
+      sha256 = "sha256-eb9lwaH5PnTjfD5b9EqSouZFdOZ4r65GQeMOHvs7bOA=";
+    }
+    {
+      url = "https://github.com/nektro/zig-detect-license";
+      rev = "de5c285d999eea68b9189b48bb000243fef0a689";
+      sha256 = "sha256-iqC1r+kPsKNkKGoF2z+93d/ryfnyYedLl5ioke/fE8U=";
+    }
+    {
+      url = "https://github.com/nektro/zig-extras";
+      rev = "01fae956e2f17aa992e717e041a3dd457d440b31";
+      sha256 = "sha256-fxhskyc+l+YIjgztrs7pQyMoV5sk9h6jS9B0nD4obJI=";
+    }
+    {
+      url = "https://github.com/nektro/zig-inquirer";
+      rev = "14c3492c46f9765c3e77436741794d1a3118cbee";
+      sha256 = "sha256-62+paPZiuyFdrV/TOZjec2y/sRgPtAvE36SLFsaSevo=";
+    }
+    {
+      url = "https://github.com/nektro/zig-json";
+      rev = "a091eaa9f9ae91c3875630ba1983b33ea04971a3";
+      sha256 = "sha256-iFrmpzD92wL+ldv/6ZKCtcNuNFkhcw56i5Yolp4xE8U=";
+    }
+    {
+      url = "https://github.com/nektro/zig-leven";
+      rev = "ab852cf74fa0b4edc530d925f0654b62c60365bf";
+      sha256 = "sha256-XUPag1aHtC+yMm0loqgHESyN8ElUWGI7GT5DLN8vZe4=";
+    }
+    {
+      url = "https://github.com/nektro/zig-licenses";
+      rev = "c9b8cbf3565675a056ad4e9b57cb4f84020e7680";
+      sha256 = "sha256-lnqDt9yaK2Ed3W14vmxcd5kraG/QDSh2rSmvC/5naiU=";
+    }
+    {
+      url = "https://github.com/nektro/zig-licenses-text";
+      rev = "3c07c6e4eb0965dafd0b029c632f823631b3169c";
+      sha256 = "sha256-Jygr1u2HYSRZNfq/aH+chaeV608dmvLIIPiFifKOWzY=";
+    }
+    {
+      url = "https://github.com/nektro/zig-range";
+      rev = "4b2f12808aa09be4b27a163efc424dd4e0415992";
+      sha256 = "sha256-Jmz2GFieXB4CALZ6XqMq9LbhkNoxF53/ovodQlg6PAM=";
+    }
+    {
+      url = "https://github.com/nektro/zig-time";
+      rev = "52a6050939b83e9f8b52b2236a36332dcb4c17a3";
+      sha256 = "sha256-g7Gmq7ZZXBB26rFezuN+lzmdm03WyPakblrkfanLPhM=";
+    }
+    {
+      url = "https://github.com/truemedian/hzzp";
+      rev = "d4fbade2d806fc93bc5f79ec8efdcc25ea625fa3";
+      sha256 = "sha256-7F92vl+5r5er36NaX/THxQ3qLRRUdAs52PW+ZeiSrZI=";
+    }
+    {
+      url = "https://github.com/truemedian/zfetch";
+      rev = "829973144f680cd16be16923041fa810c1dee417";
+      sha256 = "sha256-H3kEZ+M/OTgkC6gUjN5KHzCkd2bE5mLlES/RTT0NitI=";
+    }
+    {
+      url = "https://github.com/ziglibs/known-folders";
+      rev = "24845b0103e611c108d6bc334231c464e699742c";
+      sha256 = "sha256-xMgdOKwWqBmw7avcioqQQrrPU1MjzlBMtNjqPfOEtDQ=";
+    }
+    {
+      url = "https://github.com/yaml/libyaml";
```

Comment:
Why not to package these repos?

Reply:
What do you mean?

Reply:
As I am seeing it, each of these filesets can be installed as an independent Nixpkgs package.

Reply:
These are bound to a [lock file](https://github.com/nektro/zigmod/blob/master/zigmod.lock) and hard coded in the [build program](https://github.com/nektro/zigmod/blob/b47ad60a009653139dc811ce5c66f9b8f53cd34b/deps.zig#L51-L69).

Do you suggest adding all of them to Nixpkgs and using an override with a specific commit for each of them?

Reply:
We can pin the zig stuff but libyaml can unpinned.